sklearn/tests/test_naive_bayes.py

```diff
@@ -58,6 +58,15 @@ def test_gnb():
     assert_raises(ValueError, GaussianNB().partial_fit, X, y, classes=[0, 1])
 
 
+# TODO remove once sigma_ attribute is removed (GH #16760)
+def test_gnb_var():
+    clf = GaussianNB()
+    clf.fit(X, y)
+
+    with pytest.warns(FutureWarning):
```

Comment:
One more thing, We prefer to check the content of the warning as well: ``with pytest.warns(FutureWarning, match="Attribute sigma_ was deprecated")`

Reply:
Thanks, updated